public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -88,7 +89,12 @@ const mocks = {
 const renderRuleList = () => {
   locationService.push('/');
 
-  return render(<RuleList />);
+  return render(
+    <TestProvider>
+      <RuleList />
+    </TestProvider>,
+    { renderWithRouter: false }
+  );
```

Comment:
I've looked at this test before, and I haven't got to the bottom of why it needs `TestProvider` rather than the test utils render. For now, it seems easiest to tell the test utils not to wrap with a router, and to continue using the TestProvider (there's some code smell going on here with this test 🤔 )

Reply:
Just saw this comment after writing my own 😄  Maybe we could try applying the providers from the `test-utils` to find the ones that make this work and get rid of the `TestProvider`?